### PR TITLE
[Tests-Only] move FilesTest to pactV3 and jest

### PR DESCRIPTION
**Pull request by dpakach**
move tests to jest
- FilesTest.js

part of #645 & #646

### CODE REVIEW

File: src/helperFunctions.js [hunk 1]
```diff
@@ -310,7 +310,7 @@ class helpers {
         headers: headers
       })
         .then(async res => {
-          res.headers = new fetch.Headers(res.headers)
+          // res.headers = new fetch.Headers(res.headers)
```
Comment (dpakach) [inline src/helperFunctions.js:313]: This causes some tests to fail, Why was this added @individual-it ?

Comment (individual-it) [reply src/helperFunctions.js:313]: I think this is to have the headers as an object not array. What happens with that now? I guess we have an incompatibility between `xmlhttprequest` and `cross-fetch`

Comment (individual-it) [reply src/helperFunctions.js]: this line was added in #636 but it seems not to be needed, tests pass at the stage of #636 without it and also pass now without it
CC @LukasHirt

**Review by individual-it — APPROVED** [review]
